src/Console/Output/ProcessCommandReporter.php

```diff
@@ -44,6 +44,10 @@ public function reportLoadedRectors(): void
      */
     public function reportChangedFiles(array $changedFiles): void
     {
+        if (count($changedFiles) <= 0) {
```

Comment:
This is perfect, didn't think about that. 
:+1: for early returns

Reply:
Thanks! It's from DDD (Doman Driven Design), the responsibiltiy should be coupled to the classes that are responsible.

If sb would call this method with empty array (now knowing he or she should check for item count), it would run headline and empty list. Now it's safe and keeps responsbility from programmer in the code.